altimeter_ms5607_spi/altimeter_ms5607_spi.cc

```diff
@@ -0,0 +1,108 @@
+#include "altimeter_ms5607_spi.h"
+
+AltimeterMs5607Spi::AltimeterMs5607Spi(SPI_HandleTypeDef *hspi, GPIO_TypeDef *csPort, uint16_t csPin, GPIO_TypeDef *misoPort, uint16_t misoPin, double seaLevelPressure)
+    : _hspi(hspi), _csPort(csPort), _csPin(csPin), _misoPort(misoPort), _misoPin(misoPin), _seaLevelPressure(seaLevelPressure) {}
+
+int AltimeterMs5607Spi::Reset() {
+    HAL_GPIO_WritePin(_csPort, _csPin, GPIO_PIN_RESET);
+    // reset, opcode 0x1E
+    if (HAL_SPI_Transmit(_hspi, (uint8_t *)(const uint8_t[]){0x1E}, 1, SERIAL_TIMEOUT) != HAL_OK) return -1;
+    HAL_GPIO_WritePin(_csPort, _csPin, GPIO_PIN_SET);
+    return 0;
+}
+
+int AltimeterMs5607Spi::Init() {
```

Comment:
If this only returns two possible values for success or failure it should just return a `bool`